src/main/java/micdoodle8/mods/galacticraft/core/mixins/minecraft/ChunkProviderServerMixin.java

```diff
@@ -22,6 +22,6 @@ public class ChunkProviderServerMixin {
             require = 1)
     private boolean checkOtherModPreventGenerate(
             int chunkX, int chunkZ, World world, IChunkProvider chunkProvider, IChunkProvider chunkGenerator) {
-        return !WorldUtil.otherModPreventGenerate(chunkX, chunkZ, world, chunkGenerator, chunkProvider);
+        return !WorldUtil.otherModPreventGenerate(chunkX, chunkZ, world, chunkProvider, chunkGenerator);
```

Comment:
🤦

Reply:
😂

Reply:
To my defense: [`WorldUtil#otherModPreventGenerate`](https://github.com/GTNewHorizons/Galacticraft/blob/master/src/main/java/micdoodle8/mods/galacticraft/core/util/WorldUtil.java#L1436) lists the two last parameters the other way around and in that function they are passed to [`WorldGenerator#generate`](https://github.com/MinecraftForge/FML/blob/1.7.10/src/main/java/cpw/mods/fml/common/IWorldGenerator.java#L40) in the same order.
I don't understand *why* this change is needed, it was this swapped state before this PR too though.